backend/src/services/ecfmp.service.ts

```diff
@@ -210,4 +211,20 @@ export async function doesMeasureExist(ident: string): Promise<boolean> {
   }
 }
 
-export default { getAllMeasures, getEcfmpDetails, allocateMeasuresToPilots };
+export async function setMeasureEnable(measureId: string, enabled: boolean) {
+  try {
+    const measure: EcfmpMeasureDocument | null = await ecfmpModel.findOne({ id: measureId }).exec();
+
+    if (measure) {
+      measure.enabled = enabled;
+
+      await measure.save();
+    }
+
+    return await ecfmpModel.find().exec();
```

Comment:
Why does a function to modify a single measure return all measures?

Reply:
In order to seamlessly update the list of measures in the frontend, without having to perform a second call or filtering the old list with the new, single, measure.

Of course I can change that to whatever your likings are :)